### DISCUSSION TITLE
I guess my embedding did not work

### DISCUSSION THREAD

**Discussion by philt00n** [brianpetro/obsidian-smart-connections#796] [Q&A]
Hello guys, I am sorry for wasting anyones time in advance as I am not tech savy but loving the idea of smart connections. 
I recently installed it, added the api key for claude and I am getting a response all the time which says that 
"Based on your notes, I did not see any information about "Carl Jung's Psychology via the four stages of alchemy" or archetypal patterns of personality. Without the relevant context from your notes, I cannot determine what the first archetypal pattern of personality would be according to that framework. Please provide the specific notes you have on this topic so I can try to answer your question accurately."  "
or 
"Based on your notes, I did not see any information about the "holographic universe" concept. Your notes did not contain enough context for me to determine what important point you wanted to make about that topic. Could you please provide some additional details or context about the holographic universe idea you wanted me to address?

I guess this means that the embedding and giving context did not work. Does anyone have good sources to know if i have fixed it or not?

**Comment by brianpetro** [comment]
Does the chat show a list of notes above the response? 🌴

**Reply by philt00n** [reply]
Hi Brian, no it doesn't. It does say 
"context: undefined, undefined, undefined, undefined, undefined, undefined, undefined, undefined, und..."
Below my first question.

**Reply by brianpetro** [reply]
@philt00n that is a bug that should be fixed in the latest version. Check for updates in the community plugins tab, and let me know if updating fixes it 🌴

**Reply by philt00n** [reply]
It works now, thank you!
